### PR TITLE
RDM-9250: Identify and remove PII (Personally Identifiable Information) from CCD logs

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-9250

### Change description ###

Removed PII from application logs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java [hunk 1]
```diff
@@ -109,8 +109,8 @@ public IdamUser getUser() {
             .map(GrantedAuthority::getAuthority)
             .collect(Collectors.toSet());
 
-        String email = getUser().getEmail();
-        LOG.info("Retrieved roles for user={} roles={}", email, userRoles);
+        String userId = getUser().getId();
+        LOG.info("Retrieved roles for userId={} roles={}", userId, userRoles);
```
Comment (rafalkalita) [inline src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java:113]: Thanks for not removing this totally.

File: src/main/java/uk/gov/hmcts/ccd/auditlog/AuditService.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public void audit(AuditContext auditContext) {
         entry.setHttpMethod(auditContext.getHttpMethod());
         entry.setPath(auditContext.getRequestPath());
         entry.setRequestId(auditContext.getRequestId());
-        entry.setIdamId(userRepository.getUser().getEmail());
```
Comment (mario-paniccia) [inline src/main/java/uk/gov/hmcts/ccd/auditlog/AuditService.java:43]: maybe let's have our BA Jamal first check with the team who is processing the Audit logs whether this change is ok for them

**Review by mario-paniccia — COMMENTED** [review]
one comment with regards to removing the email from the Audit logs

**Review by smathangi — APPROVED** [review]

**Review by mario-paniccia — APPROVED** [review]
👍

File: src/main/java/uk/gov/hmcts/ccd/auditlog/AuditService.java (hunk 2 above)
Comment (mario-paniccia) [reply src/main/java/uk/gov/hmcts/ccd/auditlog/AuditService.java:43]: confirmed that this change is ok

**Pull request merged**